docs/user_guide.md

```diff
@@ -1756,10 +1757,96 @@ be compared to each other:
 CodeChecker cmd diff -p 8001 --basename my_project --newname my_new_checkin --new
 ```
 
-There is opportunity to compare a run to multiple baselines. You can simply
-provide a regular expression by `-b` flag which covers the required run names.
-The Python regex syntax has to be used:
-https://docs.python.org/2/library/re.html#regular-expression-syntax.
+**Note**: unique report identifiers are used to compare analysis results. For
+more information see [analyzer report identification](report_identification.md)
+documentation.
+
+#### Example <a name="cmd-diff-example"></a>
+Let's assume you have the following C++ code:
+```cpp
+int foo(int z)
+{
+  if (z == 0)
+    return 1 / z; // Division by zero
+
+  return 0;
+}
+
+int bar(int x)
+{
+  int y;
+  y = x % 2; // deadcode.DeadStores
+
+  return x % 2;
+}
+```
+If you analyze (`CodeChecker analyze -o ./test_report_dir
```

Comment:
Maybe a `log` example would be nice here to get the `compile_command.json` file.